ia/test/unitTest.py

```
@@ -0,0 +1,29 @@
+
+import message as msg
+
+
+class SimpleEncodingTestCase(unittest.TestCase):
+    def test_encrypt_basic(self):
+        test_strings = ["hello", "", "a", "123"]
+
+        for test_str in test_strings:
+            encrypted = msg.encrypt(test_str)
+            decrypted = msg.decrypt(encrypted)
+            self.assertEqual(test_str, decrypted, f"Failed for input: '{test_str}'")
```

Ruff (0.11.9)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)